packages/core/saved-objects/core-saved-objects-migration-server-internal/src/model/model.ts

```diff
@@ -488,7 +489,7 @@ export const model = (currentState: State, resW: ResponseType<AllActionStates>):
         // the mappings have changed, but changes might still be compatible
         return {
           ...stateP,
-          controlState: 'CHECK_UNKNOWN_DOCUMENTS',
+          controlState: 'CHECK_COMPATIBLE_MAPPINGS',
```

Comment:
That's the root change of the PR; introduction of a new step that will attempt to update the mappings (excluding the `_meta`).